pymedphys/_mosaiq/connect.py

```diff
@@ -183,7 +183,7 @@ def connect(sql_server_and_ports):
     connections, cursors = multi_connect(sql_server_and_ports)
     try:
         if len(cursors) == 1:
-            cursors = cursors[0]
+            cursors = list(cursors.values())[0]
```

Comment:
Yup, my bad on this one. Really glaringly pointing to the fact that we need a Mosaiq testing suite.

Reply:
Can I also say, I much prefer your fix over mine. Yours is much tidier. I'll make master track your fix instead of mine.